### ISSUE TITLE
Enhance SVG Accessibility with 'title' and 'desc' in Groups

### ISSUE THREAD

**Issue by mac303** [smart-byte/SwiftSVG#3]
Currently, our SVG elements lack semantic details that could improve accessibility and provide a richer context for users relying on screen readers and other assistive technologies. To address this, I propose adding `title` and `desc` elements to our SVG groups (`<g>` tags). These elements will provide textual descriptions of the graphics, enhancing the understanding of the SVG content's structure and semantics.

```
<g>
    <title>Group Title</title>
    <desc>Description of the elements within the group and their interrelation.</desc>
    <!-- SVG Elements here... -->
</g>
```

By implementing these changes, we can improve our SVGs' accessibility, making our content more inclusive and compliant with web accessibility standards. This enhancement is not only in alignment with SVG best practices but also reinforces our commitment to creating accessible content for all users.
